code-scanning/eslint.yml

```diff
@@ -0,0 +1,45 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
Please add a comment on what this workflow does

Reply:
added comments